go_expvar/datadog_checks/go_expvar/go_expvar.py

```diff
@@ -72,14 +73,15 @@ def _get_data(self, url, instance):
             'ssl_certfile': instance.get('ssl_certfile'),
             'ssl_verify': instance.get('ssl_verify'),
         }
-        for key, param in ssl_params.items():
+        ssl_params_copy = copy.copy(ssl_params)
+        for key, param in iteritems(ssl_params_copy):
```

Comment:
```suggestion
        for key, param in list(iteritems(ssl_params)):
```

We could remove the `copy` line then. Using `list()` is supposed to be slightly faster as well.